app/Http/Controllers/HerdController.php

```diff
@@ -25,7 +25,10 @@ public function edit(ElephpantsQuery $elephpantsQuery)
     public function show(string $username)
     {
         $user = User::whereUsername($username)->firstOrFail();
+        $idsElephpantsAuthenticaedUser =auth()->user()->elephpants->pluck('id')->toArray();
```

Comment:
can we name this `$authenticatedUserElephpantIds` please? and also also the `=auth()` to `= auth()` as we don't have StyleCI configured yet :D

Reply:
just enabled StyleCI with psr2 support :-)